src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import './App.css';
+import Home from './pages/Home';
+import Details from './pages/Details';
+
+function App() {
+  return (
+    <div className="App d-flex flex-column min-vh-100">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/details" element={<Details />} />
```

Comment:
- [ ] Please make sure each detail page a user opens should have a unique route. Currently, when I click on multiple countries, the routes are all the same leading to `/details` as seen in the screenshot below
![image](https://github.com/Gh-Reza/weather-app/assets/45460192/397401a6-6e75-46a6-9837-5d291f9d334d)

This route should look like this and I suggest that your parameter should be `id`
`<Route exact path="/item/:id" component={Item} />`